pytrends/request.py

```diff
@@ -9,11 +10,29 @@
 
 from pytrends import exceptions
 
+from enum import Enum
 from urllib.parse import quote
 
 
 BASE_TRENDS_URL = 'https://trends.google.com/trends'
 
+# Taken from curl_cffi broswer type enum (https://github.com/yifeikong/curl_cffi/blob/master/curl_cffi/requests/session.py)
+class BrowserType(str, Enum):
```

Comment:
But you dont have to copy and paste, just import if from: curl_cffi

from curl_cffi.requests.session import BrowserType

Reply:
Lol very true, just was quick prototyping so people could use fix asap. Couldn't convert pr to draft like I intended. 

I'll fix in next iteration. thanks